### PR TITLE
Filter config

### PR DESCRIPTION
**Pull request by CannonLock**
Closes https://github.com/PelicanPlatform/pelican/issues/654

### CODE REVIEW

**Comment by CannonLock** [comment]
@haoming29 Alright this is ready for review, copying the parameters.json was more of a adventure then it needed to be.

File: generate/param_generator.go [hunk 1]
```diff
@@ -154,7 +154,7 @@ func GenParamEnum() {
 	if err != nil {
 		panic(err)
 	}
-	// Create the json file to be generated (for the web ui)
+	// Create the json file to be generated (for the documentation website
```
Comment (haoming29) [inline generate/param_generator.go:157]:
```suggestion
	// Create the json file to be generated (for the documentation website)
```

**Review by haoming29 — CHANGES_REQUESTED** [review]
LGTM. One thing I noticed is that after I ran `make web-build`, there's a new file generated at `frontend/public/data/parameters.json` which is not added to `.gitignore`. Is this file supposed to be committed? If so, could you add it to your PR? Or it's a file for local only?

**Review by haoming29 — APPROVED** [review]
LGTM with changes

**Pull request merged**